blog/tests/admin/test_admin_image_upload_views.py

```
@@ -0,0 +1,113 @@
+
+import pytest
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.urls import reverse
+
+from blog.models import PostImage
+
+
+@pytest.mark.django_db
+def test_admin_image_upload_success(client, staff_user, content_image_file):
+    client.login(username="staffuser", password="testpass123")
```

Ruff (0.15.5)

[error] Possible hardcoded password assigned to argument: "password"

(S106)